azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CFilter.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.b2c;
+
+import lombok.Getter;
+import lombok.NonNull;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class AADB2CFilter extends OncePerRequestFilter {
+
+    @Getter
+    private final String signUpOrInRedirectURL;
+
+    private AADB2CLogoutSuccessHandler logoutSuccessHandler;
+
+    public AADB2CFilter(@NonNull AADB2CProperties b2cProperties,
+                        @NonNull AADB2CLogoutSuccessHandler logoutSuccessHandler) {
+        super();
+
+        this.logoutSuccessHandler = logoutSuccessHandler;
+        this.signUpOrInRedirectURL = b2cProperties.getPolicies().getSignUpOrSignIn().getRedirectURI();
+    }
+
+    public String getLogoutSuccessURL() {
```

Comment:
Just think of that whether we support relative path? Does Spring Security support relative path as successful logout URL?

Reply:
Yes, it support relative path, need to enhance this part for SuccessHandler.